waiter/src/waiter/process_request.clj

```diff
@@ -376,7 +378,7 @@
                           (histograms/update! (metrics/service-histogram service-id "response-size") bytes-streamed)
                           (log/error "Error occurred after streaming" bytes-streamed "bytes in response.")
                           ; Handle lower down
-                          (throw e)))]
+                          (throw (ex-info "Error occurred during streaming" {:bytes-streamed bytes-streamed} e))))]
```

Comment:
Is this related to the request log?

Reply:
Yes.  The exception handler wants `bytes-streamed` so it can report bytes streamed so far.